controllers/provisioners/eks/scaling/launchtemplate.go

```diff
@@ -311,6 +349,27 @@ func (lt *LaunchTemplate) blockDeviceList(volumes []v1alpha1.NodeVolume) []*ec2.
 	return sortTemplateDevices(devices)
 }
 
+func launchTemplateLicenseSpeficicationRequest(s []string) []*ec2.LaunchTemplateLicenseConfigurationRequest {
```

Comment:
Can we move both `launchTemplateLicenseSpeficicationRequest` and `launchTemplatePlacementRequest` into awsprovider? 
it might be more appropriate there

Reply:
doing seems to create a circular dependency:

```
package command-line-arguments
        imports github.com/keikoproj/instance-manager/api/v1alpha1
        imports github.com/keikoproj/instance-manager/controllers/providers/aws
        imports github.com/keikoproj/instance-manager/controllers/provisioners/eks/scaling
        imports github.com/keikoproj/instance-manager/api/v1alpha1: import cycle not allowed
package command-line-arguments
        imports github.com/keikoproj/instance-manager/api/v1alpha1
        imports github.com/keikoproj/instance-manager/controllers/providers/aws
        imports github.com/keikoproj/instance-manager/controllers/provisioners/eks/scaling
        imports github.com/keikoproj/instance-manager/controllers/providers/aws: import cycle not allowed
```

Reply:
This is because of `LaunchTemplatePlacementInput` which is part of scaling package, suggest you remove that struct and just pass in the input you need as arguments.

e.g.
```go
func (w *AwsWorker) GetLaunchTemplatePlacementRequest(az, arn, tenancy string) {
  ...
}
```